src/js/apps/patients/patient/sidebar/sidebar_app.js

```diff
@@ -1,117 +1,30 @@
-import Backbone from 'backbone';
 import Radio from 'backbone.radio';
 
 import App from 'js/base/app';
 
+import SidebarWidgetsApp from 'js/apps/patients/patient/sidebar/sidebar-widgets_app';
 import { SidebarView } from 'js/views/patients/patient/sidebar/sidebar_views';
 
 export default App.extend({
+  childApps: {
+    widgets: SidebarWidgetsApp,
+  },
   onBeforeStart({ patient }) {
-    const currentOrg = Radio.request('bootstrap', 'currentOrg');
-
-    this.orgEngagementEnabled = currentOrg.getSetting('engagement');
     this.patient = patient;
 
-    const widgets = this.getSidebarWidgets();
-
     this.showView(new SidebarView({
       model: this.patient,
-      collection: widgets,
     }));
+
+    this.startChildApp('widgets', {
+      region: this.getRegion('widgets'),
+      patient: this.patient,
+    });
   },
   beforeStart({ patient }) {
-    return Radio.request('entities', 'fetch:patient:engagementStatus', patient.id);
+    return [Radio.request('entities', 'fetch:patient:engagementStatus', patient.id)];
```

Comment:
I think we should move this request and the `trigger('status...` into the widget `initialize` for now.  For now this widget is always the gate keeper to this info.  I don't typically like data requests in views, but these widgets are a grey area.  So it'd be `Radio.request(...).fail(() => { trigger`. 

And this app's `beforeStart` could be used for the widget request.. so we don't really need the childapp.  However..  I'm also thinking that until we have a better caching strategy for these long lasting configs like widgets.. I think we should probably just load every widget (no filter on the request) at the bootstrap vs introducing loaders everywhere.  If we ever get out of hand with the number of widgets we can readdress.

Reply:
This makes sense.  We'll probably have a few norm-breaking instances like this in these widgets. Added bonus: moving the request to the widget allowed me to remove the delay on the request stub in the test 🎉